electrumsv/nodeapi.py

```diff
@@ -163,16 +165,25 @@ async def shutdown_async(self) -> None:
         assert self._runner is not None
         await self._runner.cleanup()
 
-    def event_transaction_added(self, transaction_hash: bytes) -> None:
+    def event_transaction_change(self, transaction_hash: bytes) -> None:
         """
-        Called by `Wallet._import_transaction_async` as an explicit JSON-RPC API hook.
+        Called by various wallet code as an explicit JSON-RPC API hook.
+
+        - When a transaction is added.
+        - When a transaction is verified (mined or post-reorg).
 
         This is non-blocking and launches a thread to run the notification command in. The user
         can pass multiple commands to run for each notification, and while the node only allows
         one it doesn't really make any difference and it's more work to try and limit the
         command-line argument passing than just handle it.
         """
 
+        # nocheckin
```

Comment:
I'm guessing this is not intended to be left in the commit?

Reply:
No, but actually I am okay with it. It helps illustrate the callbacks without having to set up a callback and highlights the code should be removed at some point.

Reply:
Okay. Otherwise all looks good.